### PR TITLE
Mark some common project settings as basic

### PR DESCRIPTION
**Pull request by Calinou**
This marks advanced settings that were agreed upon in https://github.com/godotengine/godot/issues/50620 as basic.

### CODE REVIEW

**Review by AaronRecord — APPROVED** [review]

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -4929,7 +4929,7 @@ bool EditorNode::is_run_playing() const {
 String EditorNode::get_run_playing_scene() const {
 	String run_filename = editor_run.get_running_scene();
 	if (run_filename.is_empty() && is_run_playing()) {
-		run_filename = GLOBAL_DEF("application/run/main_scene", ""); // Must be the main scene then.
+		run_filename = GLOBAL_DEF_BASIC("application/run/main_scene", ""); // Must be the main scene then.
```
Comment (akien-mga) [inline editor/editor_node.cpp:4932]: Settings shouldn't be defined twice. And I feel EditorNode is not the right place to define this, it's used and should be defined in Main IMO.

**Review by fire — APPROVED** [review]
The list looks ok. No comments on the settings being defined twice. Probably should be split into a separate proposal.

**Comment by akien-mga** [comment]
> No comments on the settings being defined twice. Probably should be split into a separate proposal.

Fair enough, there's quite a few in that situation.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**